indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexTuningConfig.java

```diff
@@ -43,6 +42,8 @@
 
   private static final Duration DEFAULT_CHAT_HANDLER_TIMEOUT = new Period("PT10S").toStandardDuration();
   private static final int DEFAULT_CHAT_HANDLER_NUM_RETRIES = 5;
+  private static final int DEFAULT_MAX_NUM_SEGMENTS_TO_MERGE = 100;
+  private static final int DEFAULT_TOTAL_NUM_MERGE_TASKS = 10;
```

Comment:
Could consider making the default number of merge tasks the same as the effective value for `maxNumSubTasks` instead

Reply:
`maxNumSubTasks` is actually the number of max number of sub tasks that can be run concurrently at the same time. Raised https://github.com/apache/incubator-druid/issues/8318.